integration.cloudbuild.yaml

```
@@ -74,3 +82,4 @@ substitutions:
   _GCR_HOSTNAME: ${_REGION}-docker.pkg.dev
   _SERVICE: demo-service-${BUILD_ID}
   _REGION: us-central1
+  _TOOLBOX_URL: https://toolbox-service-706535509072.us-central1.run.app
```

```suggestion
  _TOOLBOX_URL: https://toolbox-service-${PROJECT_NUMBER}.us-central1.run.app
```

Or we can do `_TOOLBOX_URL: http://127.0.0.1:5000`, and then add a env var substitution within the cloud build trigger.